docs/features/correlation.md

```diff
@@ -5,6 +5,8 @@ layout: default
 
 # Correlation Between HTTP Requests
 
+![](https://img.shields.io/badge/Available%20starting-v0.4-green?link=https://github.com/arcus-azure/arcus.webapi/releases/tag/v0.4.0)
```

Comment:
Frankly, I don't really care about this if I'm viewing the latest version, no?

Reply:
Yes, I'll delete them.